06-reproj.Rmd

```diff
@@ -455,7 +455,7 @@ unique(cat_raster)
 
 When reprojecting categorical rasters, the estimated values must be the same as those of the original.
 This could be done using the nearest neighbor method (`ngb`).
-This method assigns new cell values to the nearest cell center of the input raster.
+This method sets each new cell value to the value of the nearest cell (center) of the input raster.
```

Comment:
Definite improvement. I would go further and combine with the previous sentence, e.g.:

```
This could be done using the nearest neighbor method (`ngb`), which sets reprojected cell values to the value of the nearest cell in the input raster.
```

Not sure the words `new` or `(center)` are needed.

Thoughts @jannes-m and @Nowosad ? Overall I'm happy with the suggested change but suspect it could be further improved on a second (or perhaps 5th) reading.